### ISSUE TITLE
Seems to be an issue with expanding UUID value

### ISSUE THREAD

**Issue by vicmosin** [damnhandy/Handy-URI-Templates#34]
I am facing the following issue: once I pass an UUID value into parameters map and expands the template, the final result contains the UUID's internal properties such as `mostSigBits` and `leastSigBits` instead of simple string representation. Converting the UUID to string before passing to `expand` method solves the problem

**Comment by damnhandy** [comment]
Can you post the output you're seeing? I think I know what might be going on. This would likely address issues like #26 and others.

**Comment by vicmosin** [comment]
```
http://localhost:7070/chats?id=-7375646386867656971,7078318184015086714
```

original uri: `http://localhost:7070/chats{?page,size,sort,filter,id}`

**Comment by damnhandy** [comment]
Fixed in the `2.1.0-SNAPSHOT` if you could give it a whirl and see if verify if the behavior is what you're expecting, i'll close this issue.

**Comment by vicmosin** [comment]
Confirmed, `2.1.0-SNAPSHOT` fixes the issue
